plugin.googledrive/addon.py

```diff
@@ -0,0 +1,201 @@
+#-------------------------------------------------------------------------------
```

Comment:
usually a good practice to keep your addon entrypoints as small as possible since they are the only files that are not compiled to bytecode. Create a dummy entrypoint file and import and run this one instead:

```
from resources.lib.addon import addon
addon.start()
```

Reply:
Changed